kvmagent/kvmagent/plugins/host_pushgateway.py

```
@@ -0,0 +1,48 @@
+        conf_paths = []
+        for pattern in LIGHTTPD_CONF_GLOBS:
+            conf_paths.extend(glob.glob(pattern))
+
+    for conf_path in conf_paths:
+        try:
+            with open(conf_path, "r") as fd:
+                match = AUTH_HEADER_PATTERN.search(fd.read())
+                if match:
+                    return match.group(1)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)